database/README.md

```diff
@@ -411,6 +411,18 @@ order\_uids | bytea[] | not null | order uids that were included in the auction
 Indexes:
 - PRIMARY KEY: btree(`auction_uid`)
 
+### auction\_surplus\_capturing\_jit\_order\_owners
+
+Stores all surplus capturing jit order owners that are part of an auction. Surplus capturing jit order is allowed to be settled in an auction only if it's owner was part of that auction.
```

Comment:
IMO it's not strictly forbidden to settle these JIT orders it's just that there really is not reason for a solver to do that since they don't count towards surplus.

```suggestion
Stores all surplus capturing jit order owners that are part of an auction. JIT orders settled for addresses which were not part of a given auction will not count towards surplus.
```